products/ocp4/profiles/stig-node.profile

```diff
@@ -21,5 +21,7 @@ description: |-
 
 filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "ocp4-node-on-sdn" in platforms or "ocp4-node-on-ovn" in platforms'
 
+extends: stig-node-v1r1
+
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.